src/hub/Hub.sol

```diff
@@ -233,14 +233,7 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
         (uint128 approvedAssetAmount,) =
             shareClassManager.approveDeposits(unlockedPoolId, scId, maxApproval, paymentAssetId, valuation);
 
-        uint128 valueChange = holdings.increase(unlockedPoolId, scId, paymentAssetId, valuation, approvedAssetAmount);
-
-        accounting.addCredit(
-            holdings.accountId(unlockedPoolId, scId, paymentAssetId, uint8(AccountType.Equity)), valueChange
-        );
-        accounting.addDebit(
-            holdings.accountId(unlockedPoolId, scId, paymentAssetId, uint8(AccountType.Asset)), valueChange
-        );
+        sender.sendApprovedDeposits(unlockedPoolId, scId, paymentAssetId, approvedAssetAmount);
```

Comment:
Note: This PR makes CV the single provider of holding updates. When it receives `ApprovedDeposits`, it dispatches the holding update. It's clear this round trip is more costly but it comes with a couple of business side advantages ([more info](https://kflabs.slack.com/archives/C07PG2EUR9C/p1743689530352989?thread_ts=1743669923.076539&cid=C07PG2EUR9C)).